src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-controller.php

```diff
@@ -290,6 +294,7 @@ public function update_item( $request ) {
 	 * Prepares a single global styles config for update.
 	 *
 	 * @since 5.9.0
+	 * @since 6.2.0 Added validation of styles.css property.
 	 *
 	 * @param WP_REST_Request $request Request object.
 	 * @return stdClass Changes to pass to wp_update_post.
```

Comment:
The method now will also return `WP_Error`. The `@return` needs to include this:

```php
* @return stdClass|WP_Error Prepared item on success. WP_Error on when the custom CSS is not valid.
```

Reply:
Fixed.